### DISCUSSION TITLE
bowtie2 tutorial error

### DISCUSSION THREAD

**Discussion by IceGreb** [sandbox-bio/sandbox.bio#8] [Q&A] [answered]
Greetings and thank you for this amazingly helpful tool!

While running the bowtie2 tutorial and in the bcftools variant calling section i encountered the following :

![error1](https://user-images.githubusercontent.com/127413808/224487743-44749ff4-82b5-4ead-8b53-2fecdaa58de8.jpg)

*i also tried to give the direct path of the .fasta

**Comment by robertaboukhalil** [comment, marked as answer]
Thanks @IceGreb!

That's strange, your `REF` and `REF_FASTA` look a little different than they should, yours are missing the `/shared/` prefix 🤔 

Can you try running the following:

```bash
REF=/shared/bowtie2/example/index/lambda_virus
REF_FASTA=/shared/bowtie2/example/reference/lambda_virus.fa
```

And retrying?

**Reply by IceGreb** [reply]
It worked! :smiley:  Thanks!
